### DISCUSSION TITLE
Unable to complete the tax form in the GitHub Sponsors

### DISCUSSION THREAD

**Discussion by andrzejkupczyk** [community/community#22805] [New to GitHub]
<p>This is what Chrome throws at me in the console when I click the “Complete the tax form” link:</p>
<blockquote>
<p>Refused to send form data to ‘<a href="https://na.account.docusign.com/" rel="noopener nofollow ugc">https://na.account.docusign.com/</a>’ because it violates the following Content Security Policy directive: “form-action ‘self’ <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> <em>[more domains here]</em>”.</p>
</blockquote>
<p>Is it a good place to report this problem?</p>
<p>Ubuntu 20.04.1<br>
Chrome version 85.0.4183.83</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>We’ve had a few other GitHub users report this issue, and our Sponsors team is investigating :mag: .</p>
<p>In the meantime, we’ve heard from others that they’ve managed to access this by temporarily enabling pop-ups or using another browser such as Safari or Firefox - If you’re able to try one of these approaches, you might find you’re able to get to the document (and submit your application) sooner.</p>

**Comment by andrzejkupczyk** [comment]
<p>Firefox FTW, thank you :bowing_man:</p>
